.editorconfig

```diff
@@ -0,0 +1,29 @@
+# http://EditorConfig.org
+
+# This file is the top-most EditorConfig file
+root = true
+
+# All Files
+[*]
+charset = utf-8
+end_of_line = lf
+indent_style = space
+indent_size = 4
```

Comment:
How about using tabs with size of 2?

Reply:
As of now all files have tab size 4, so all of them will be changed after reformatting